src/main/java/pico/inference/PICOInferenceRealTypeFactory.java

```diff
@@ -128,19 +125,19 @@ public boolean getShouldDefaultTypeVarLocals() {
     }
 
     // Copied from PICOAnnotatedTypeFactory
-    @Override
-    protected void annotateInheritedFromClass(AnnotatedTypeMirror type, Set<AnnotationMirror> fromClass) {
```

Comment:
This method is removed, so there is nothing to override.

For now, still need to find a way to implement this by using new method.